src/diagram/components/variable-form.tsx

```diff
@@ -1,80 +0,0 @@
-import { observer } from "mobx-react-lite";
-import React from "react";
-import { Operation, VariableType } from "../models/variable";
-
-interface IProps {
-    variable: VariableType;
-}
-  
-const _VariableForm: React.FC<IProps> = ({ variable }) => {
-  const onValueChange = (evt: any) => {
-    // if the value is null or undefined or empty just store undefined
-    if (evt.target.value == null || evt.target.value === "") {
-      variable.setValue(undefined);
-    } else {
-      // If the value is the empty string parseFloat turns that into NaN which
-      // we want to avoid.
-      variable.setValue(parseFloat(evt.target.value));
-    }
```

Comment:
@eireland this was something I changed that you should move over to the quantity-node.tsx. Basically it is checking for an `""` value as well.